src/document.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Sajari;
+
+class Key
+{
+    /** @var $field string */
+    private $field;
+    private $value;
+
+    /**
+     * Key constructor.
+     * @param $field string
+     * @param $value
+     */
+    public function __construct($field, $value)
+    {
+        $this->field = $field;
+        $this->value = $value;
+    }
+
+    /**
+     * @return string
+     */
+    public function getField()
+    {
+        return $this->field;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getValue()
+    {
+        return $this->value;
+    }
+
+    /**
+     * @return \sajari\engine\store\doc\Keys\Key
```

Comment:
@tbillington you'd need to check, but PHP namespaces with upper and lower case may have issues. `Sajari` and `sajari` I mean. Is it possible to have the auto-gen'd proto stuff use namespace `Sajari`? Further, can it make the namespaces first letter uppercase?

It probably wouldn't be a problem if the proto stuff was never exposed to the public API, but it looks like in this method for example, that it is exposed.